objects_list: [HV8_0]
